dotnet/src/IntegrationTests/Connectors/Memory/Qdrant/QdrantVectorStoreTests.cs

```diff
@@ -8,19 +8,24 @@ namespace SemanticKernel.IntegrationTests.Connectors.Memory.Qdrant;
 
 [Collection("QdrantVectorStoreCollection")]
 public class QdrantVectorStoreTests(QdrantVectorStoreFixture fixture)
+#pragma warning disable CA2000 // Dispose objects before losing scope
     : BaseVectorStoreTests<ulong, QdrantVectorStoreFixture.HotelInfo>(new QdrantVectorStore(fixture.QdrantClient))
+#pragma warning restore CA2000 // Dispose objects before losing scope
 {
     [Fact]
     public async Task ItPassesSettingsFromVectorStoreToCollectionAsync()
     {
+        // The client is shared with base class tests.
+        const bool OwnsClient = false;
```

Comment:
nit: any reason to not inline?

Reply:
It's used in two places, having the const allows me to have a single comment for it.